### ISSUE TITLE
Unable to Resize Grid Columns

### ISSUE THREAD

**Issue by joshgk00** [vanjarosoftware/Vanjaro.Platform#25]
I added a Grid to a section. I tried to resize the column by dragging and dropping but it just reverts back to the original predefined setting that was chosen when adding the grid. Here's a screencast of what happened.
https://www.screencast.com/t/oRrSTrRJZ

**Comment by meetmandeep** [comment]
I'm unable to duplicate this on a new page. 

From Menu > Site > Pages, select the page in question and "Save Page as Template". Give it a name. 

You'll find this saved at /Portals/PortalID/vThemes/Default/templates/pages/ 

Please find and attach that .json file here so we can replicate it.

**Comment by joshgk00** [comment]
Here's the .json file from the page.
[Home-Grid Issue.zip](https://github.com/vanjarosoftware/Vanjaro.Platform/files/4389995/Home-Grid.Issue.zip)

**Comment by meetmandeep** [comment]
I'm unable to reproduce with that template. Does the issue persist for you on every page? All grids? Or just that particular one?

**Comment by joshgk00** [comment]
I tried it on another page and had the experience. The grid didn't resize.  The site is installed locally via nvQuickSite. I also installed the latest version of Action Forms. That's the only app installed.

**Comment by meetmandeep** [comment]
What browser/version you're using? What OS? What's the current desktop resolution? Thanks

**Comment by joshgk00** [comment]
Browser: Brave 1.5.115, Chromium 80.0.3987
Windows 10
Resolution 3840x2160

I think I figured it out. The browser wasn't full screen Because of the large resolution so the drag and drop resizing wasn't working. When I went full screen with the browser the grids worked as expected. So I'm not sure if this is a true bug or not. The browser window was still large enough to be in desktop mode but I'm sure it was close to changing to tablet.
